### ISSUE TITLE
Index creation failure on 1.2.120

### ISSUE THREAD

**Issue by GoogleCodeExporter** [sagarswathi/h2database#133]
```
I am getting some strange errors when switching from Version 1.2.119 to 
1.2.120 with the database:

General error: java.lang.RuntimeException: Wrong index: 
EXCEPTIONLIST_TABLE_SCAN:520 PRODUCTTEXT_TABLE_SCAN:550; SQL statement:
create unique index producttext_pk on producttext(productid, basetextid) 
[50000-120] HY000/50000

After this thing happens to a table, the table is not accessible anymore. 
```

Original issue reported on code.google.com by `mborcher...@gmail.com` on 6 Oct 2009 at 9:45

**Comment by GoogleCodeExporter** [comment]
```
Thanks for reporting! I think I found the problem, it will be fixed in the next
release (soon, in a few days). If you want to try now, you would need to build 
H2
yourself.
```

Original comment by `thomas.t...@gmail.com` on 7 Oct 2009 at 7:10
- Changed state: **Accepted**

**Comment by GoogleCodeExporter** [comment]
```
This should be fixed in version 1.2.121
```

Original comment by `thomas.t...@gmail.com` on 11 Oct 2009 at 7:30
- Changed state: **Fixed**

**Comment by GoogleCodeExporter** [comment]
```
It works now! The new page store requires much less space! Performance seems to 
be
also a littler better... very fast fix... H2 rocks :-)
```

Original comment by `mborcher...@gmail.com` on 13 Oct 2009 at 5:54
